packages/frontend/src/utils/formatters/padZeroes.ts

```diff
@@ -0,0 +1,7 @@
+import { BigNumber } from '@ethersproject/bignumber'
+
+export function padZeroes(number: BigNumber | number, length = 2) {
+  const numberString = number.toString()
+  const missingLength = length - numberString.length
+  return missingLength > 0 ? '0'.repeat(missingLength) + numberString : numberString
+}
```

Comment:
Isn't it the same as String.padStart()?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

Reply:
It is ;_;